thingsboard.advisories.yaml

```diff
@@ -87,6 +87,10 @@ advisories:
             componentType: java-archive
             componentLocation: /usr/share/tb-mqtt-transport/bin/tb-mqtt-transport.jar
             scanner: grype
+      - timestamp: 2024-09-15T09:19:20Z
+        type: pending-upstream-fix
+        data:
+          note: This vulnerability requires upstream changes to support the non-vulnerable library.
```

Comment:
What kind of changes?

Reply:
I am gonna rely on these changes https://github.com/wolfi-dev/os/pull/28209 to fix this and another CVE 🤞🏻 .